values.yaml

```
@@ -728,6 +728,7 @@ csi:
     # YAML-formatted multi-line templated string map of the annotations to apply
     # to the daemonSet.
     annotations: {}
+    tolerations: null
```

I think this could use a comment block describing format, etc. Something like this?

```suggestion

    # Toleration Settings for provider pods
    # This should be a multi-line string matching the Toleration array
    # in a PodSpec.
    tolerations: null

```